elasticsearch/src/main/java/com/gentics/mesh/search/index/group/GroupIndexHandlerImpl.java

```diff
@@ -72,10 +72,10 @@ public GroupMappingProvider getMappingProvider() {
 	}
 
 	@Override
-	public Map<String, IndexInfo> getIndices() {
+	public Map<String, Optional<IndexInfo>> getIndices() {
```

Comment:
The main point is to replace index creation with `IndexInfo` with an `Optional<IndexInfo>`, meaning the presence as creation, and the absence as removal/ignorance.